agentforce/1.1/modules/ROOT/pages/agentforce-connector-reference.adoc

```diff
@@ -0,0 +1,501 @@
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Any
+| Attributes Type a| <<InvokeAgentResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* MS-AGENTFORCE:AGENT_METADATA_FAILURE 
+* MS-AGENTFORCE:AGENT_OPERATIONS_FAILURE 
+* MS-AGENTFORCE:CONNECTIVITY 
+* MS-AGENTFORCE:RETRY_EXHAUSTED 
+
+
+[[Unauthorize]]
+== Unauthorize
+`<ms-agentforce:unauthorize>`
+
+Deletes all the access token information of a given resource owner ID so that it's impossible to execute any operation for that user without doing the authorization dance again
+
+=== Parameters
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+== Types
+
+[[Tls]]
+=== Tls
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | Comma-separated list of protocols enabled for this context. |  | 
+| Enabled Cipher Suites a| String | Comma-separated list of cipher suites enabled for this context. |  | 
+| Trust Store a| <<TrustStore>> |  |  | 
+| Key Store a| <<KeyStore>> |  |  | 
+| Revocation Check a| * <<StandardRevocationCheck>>
+* <<CustomOcspResponder>>
+* <<CrlFile>> |  |  | 
+|======================
+
+[[TrustStore]]
+=== Trust Store
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the truststore. |  | 
+| Password a| String | Password that protects the truststore. |  | 
+| Type a| String | Type of truststore. |  | 
+| Algorithm a| String | Encryption algorithm that the truststore uses.|  | 
+| Insecure a| Boolean | If `true`, Mule stops performing certificate validations. Setting this to `true` can make connections vulnerable to attacks. |  | 
+|======================
+
+[[KeyStore]]
+=== Key Store
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | Path to the keystore. Mule resolves the path relative to the current classpath and file system, if possible. |  | 
+| Type a| String | Type of keystore to use. |  | 
+| Alias a| String | Alias of the key to use when the keystore contains multiple private keys. By default, Mule uses the first key in the file. |  | 
+| Key Password a| String | Password used to protect the private key. |  | 
+| Password a| String | Password used to protect the keystore. |  | 
+| Algorithm a| String | Encryption algorithm that the keystore uses. |  | 
+|======================
+
+[[StandardRevocationCheck]]
+=== Standard Revocation Check
```

Comment:
the formatting for this type isn't rendering. You can copy paste the type from here

https://docs.mulesoft.com/salesforce-pubsub-connector/latest/salesforce-pubsub-connector-reference#StandardRevocationCheck